sagemaker_batch_transform/tensorflow_open-images_tfrecord/tensorflow-serving-tfrecord-python-sdk.ipynb

```diff
@@ -61,12 +61,18 @@
   {
```

Comment:
<p><code>iterator = tf.compat.v1.python_io.tf_record_iterator('train-00001-of-00100')</code>  is causing CI problem, because <code>train-00001-of-00100</code>  is in <code>download_path</code>.  Try <code>iterator = tf.compat.v1.python_io.tf_record_iterator(os.path.join('download_path', 'train-00001-of-00100'))</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1906/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>